api/client/lib.py

```diff
@@ -513,6 +516,8 @@ def list_of_series_to_single_series(series_list, add_belongs_to=False):
                 'frequency_id': series['series'].get('frequencyId', None)
                 # 'source_id': series['series'].get('sourceId', None), TODO: add source to output
             }
+            if series_metadata:
```

Comment:
If this metadata's included, it will break the pandas `get_df()` function again like we had with the `'belongsTo'` property https://github.com/gro-intelligence/api-client/pull/147/files. Can't hash the dict values. So, another reason to exclude metadata for now like I said in my other comment.

Reply:
removed from metadata from point